woocommerce/woocommerce-gutenberg-products-block
# Add Cart Summary block in cart i2

Adds Order Summary block (Not sure if it should be named Order Summary? still).

This block only includes line items like subtotal, shipping, coupons, totals... and doesn't include the express payment buttons or continue to checkout button, those are separate blocks.

Part of #4709, based on #4780

Testing Steps
Insert Cart i2.
Make sure you can see the sidebar.
Make sure it's using preview data.
